application/src/integrationTest/kotlin/org/gxf/standalonenotifyinggateway/coaphttpproxy/IntegrationTestCoapClient.kt

```diff
@@ -30,6 +30,8 @@ class IntegrationTestCoapClient {
 
     @Value("\${config.psk.default-key}") private lateinit var defaultKey: String
 
+    @Value("\${config.coap.cipher-suites}") private lateinit var cipherSuites: List<String>
```

Comment:
I think this should also work:
```suggestion
    @Value("\${config.coap.cipher-suites}") private lateinit var cipherSuites: List<CipherSuite>
```